apps/OpenSpace/ext/launcher/src/sgctedit/windowcontrol.cpp

```diff
@@ -297,35 +334,52 @@ void WindowControl::createWidgets(const QColor& windowColor) {
             "images available to other real-time applications on the same machine for "
             "further processing"
         );
-        layout->addWidget(_spoutOutput);
+        layout->addWidget(_spoutOutput, 6, 0, 1, 4);
     }
+
+    //
+    // Render 2D & 3D
     {
-        QFrame* projectionGroup = new QFrame;
-        projectionGroup->setVisible(true);
-        projectionGroup->setFrameStyle(QFrame::StyledPanel | QFrame::Plain);
+        _render2D = new QCheckBox("Render Overlays");
+        _render2D->setToolTip(
+            "Determines whether any overlays should be\nrendered on this window. "
+            "Overlays in this case are\nthe user interface, dashboard information, and "
+            "other\nelements that are only useful for a pilot."
+        );
+        layout->addWidget(_render2D, 5, 4, 1, 4);
+
+        _render3D = new QCheckBox("Render Scene");
+        _render3D->setToolTip(
+            "Determines whether the main 3D scene should be\nrendered on this window. "
+            "This contains anything that is not the overlay\nlike 3D models, the "
```

Comment:
A bit weird to compare with the overlay here, IMO. Consider rewriting or removing the second sentence